src/Microsoft.Health.Fhir.R4.Web/Program.cs

```diff
@@ -0,0 +1,40 @@
+﻿// -------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
+// -------------------------------------------------------------------------------------------------
+
+using Microsoft.AspNetCore;
+using Microsoft.AspNetCore.Hosting;
+using Microsoft.Azure.KeyVault;
+using Microsoft.Azure.Services.AppAuthentication;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.Configuration.AzureKeyVault;
+
+namespace Microsoft.Health.Fhir.Web
+{
+    public static class Program
```

Comment:
Can this also be shared?

Reply:
Looks like it can. An earlier version I had it failed to find the main method, but I seemed to have resolved that separately.